samples/client_campaigns_sample.rb

```diff
@@ -0,0 +1,56 @@
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
+require 'createsend'
+
+class ClientCampaignsSample
+    def initialize
+        raise 'CREATESEND_API_KEY env var missing' if ENV['CREATESEND_API_KEY'].nil?
+        raise 'CREATESEND_CLIENT_ID env var missing' if ENV['CREATESEND_CLIENT_ID'].nil?
+
+        auth = {:api_key => ENV['CREATESEND_API_KEY']}
+        @client = CreateSend::Client.new auth, ENV['CREATESEND_CLIENT_ID']
+    end
+
+    def get_all_sent_campaigns
+        campaigns = []
+        page_number = 1
+        loop do
+            page = @client.campaigns(page_number, 1000, 'desc', '', '', '')

```

Comment:
Considering adding some values here if we want to demon full usage of all the filter parameters?
Ex. Add the demo format for how to pass through tag array? Ex. "tag1,tag2"
Add some sample value for date filter in YYYY-MM-DD format to demonstrate the correct format for passing through date value?

Reply:
I do have some samples at the bottom of this class, however I'll move them up just so it's more obvious it's connected to this method

Reply:
Sorry I miss it, just one last suggestion, 
Can we add a function for get_campaigns_with_tags?
In that case, we could demonstrate how to pass in a tag array.